### PR TITLE
Session share

### LINKED ISSUE #508: Session share
## Kind (Required)

- Improvement

 ## Overview (Required)

Implement the sharing function.

Screenshot_20200121-153807

Example URL

```
https://droidkaigi.jp/2020/timetable/155510
https://droidkaigi.jp/2020/timetable/156392
https://droidkaigi.jp/2020/en/timetable/155510
https://droidkaigi.jp/2020/en/timetable/156392
```

 ## Links	

-

### PR DESCRIPTION
**Pull request by kako351**
## Issue
- close #508 

## Overview (Required)
- Implement the sharing function.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/session/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -12,4 +12,5 @@
     <string name="query_hint">検索する</string>
     <string name="slide">スライド</string>
     <string name="empty_favorite_session">お気に入りのセッションを登録するとあなた専用のプランを作ることができます。</string>
-</resources>
\ No newline at end of file
+    <string name="session_share_url">https://droidkaigi.jp/2020/timetable/%d</string>
```
Comment (takahirom) [inline feature/session/src/main/res/values-ja/strings.xml]: 👍

**Comment by takahirom** [comment]
Sorry, Please fix danger issues 🙇

**Comment by kako351** [comment]
Sorry I fix not yet.I will do it on the weekend.

**Comment by kako351** [comment]
Fixed dangerous issues and resolved conflicts. Please review again. 🙏

**Pull request merged**